### PR TITLE
Add temporary description of how to use UCN with Custom Config

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

**Review by hasancelik — APPROVED** [review]

**Comment by hasancelik** [comment]
@hazelcast/documentation could you backport this PR to 5.11 branch?

File: docs/modules/ROOT/pages/user-code-namespaces.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast
+spec:
+  licenseKeySecretName: hazelcast-license-key
+  customConfigCmName: custom-config-cm
+----
+
+With the User Code Namespaces enabled, you can create and use it from the client as described in xref:hazelcast:clusters:ucn-dynamic-client.adoc[Configure Client].
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/user-code-namespaces.adoc:33]:
```suggestion
With User Code Namespaces enabled, you can create and use the namespaces from the client as described in xref:hazelcast:clusters:ucn-dynamic-client.adoc[Configure Client].
+
NOTE: The same User Code Namespaces must have been configured in your data structure.
```

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]
One suggestion.

I added the note due to a Slack discussion earlier today (https://hazelcast.slack.com/archives/C035HQET5/p1713272907972899).

File: docs/modules/ROOT/pages/user-code-namespaces.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+kind: Hazelcast
+metadata:
+  name: hazelcast
+spec:
+  licenseKeySecretName: hazelcast-license-key
+  customConfigCmName: custom-config-cm
+----
+
+With User Code Namespaces enabled, you can create and use the namespaces from the client as described in xref:hazelcast:clusters:ucn-dynamic-client.adoc[Configure Client].
++
```
[the remark this answers is not in the document]
Comment (SeriyBg) [reply docs/modules/ROOT/pages/user-code-namespaces.adoc]: No worries, accepted the suggestion 😃

**Comment by rebekah-lawrence** [comment]
Thanks - I'm just sorting out the backporting requested by Hasan then this will be ready to go

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Pull request merged**